protocol/x/clob/keeper/process_single_match.go

```diff
@@ -243,6 +246,25 @@ func (k Keeper) ProcessSingleMatch(
 			notionalLiquidatedQuoteQuantums,
 			takerInsuranceFundDelta,
 		)
+
+		labels := []gometrics.Label{
+			metrics.GetLabelForIntValue(metrics.PerpetualId, int(perpetualId)),
+			metrics.GetLabelForBoolValue(metrics.IsBuy, matchWithOrders.TakerOrder.IsBuy()),
+			metrics.GetLabelForBoolValue(metrics.CheckTx, ctx.IsCheckTx()),
+		}
+
+		// Stat quote quantums liquidated.
+		telemetry.IncrCounterWithLabels(
+			[]string{metrics.Liquidations, metrics.PlacePerpetualLiquidation, metrics.Filled, metrics.QuoteQuantums},
+			metrics.GetMetricValueFromBigInt(notionalLiquidatedQuoteQuantums),
+			labels,
+		)
+		// Stat insurance fund delta.
+		telemetry.IncrCounterWithLabels(
+			[]string{metrics.Liquidations, metrics.InsuranceFundDelta},
+			metrics.GetMetricValueFromBigInt(new(big.Int).Abs(takerInsuranceFundDelta)),
+			append(labels, metrics.GetLabelForBoolValue(metrics.Positive, takerInsuranceFundDelta.Sign() == 1)),
+		)
```

Comment:
when you use a counter, is it possible to get a nice graph where you can see that $100k got liquidated in this hour and $200k got liquidated in the next hour? Or are you stuck with having only cumulative-sum graphs?

Reply:
I think you will need to zoom in to this hour to see the cumulative sum. then zoom in to next hour to see the cumulative sum for the next hour.